src/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -70,16 +70,22 @@ internal override async Task<AuthenticationResult> ExecuteAsync(CancellationToke
 
             if (AuthenticationRequestParameters.IsBrokerEnabled)
             {
+                BrokerFactory brokerFactory = new BrokerFactory();
                 var brokerInteractiveRequest = new BrokerInteractiveRequest(
                     AuthenticationRequestParameters,
                     _interactiveParameters,
                     ServiceBundle,
-                    _authorizationResult);
+                    _authorizationResult,
+                    brokerFactory.Create(ServiceBundle));
 
                 if (brokerInteractiveRequest.IsBrokerInvocationRequired())
                 {
                     _msalTokenResponse = await brokerInteractiveRequest.SendTokenRequestToBrokerAsync().ConfigureAwait(false);
```

Comment:
No cancellation support?

Reply:
I'd say that this can be a bug that we need to look at separately. Cancelling is not natively supported by the iOS Dispatch Queue so it won't be trivial to implement. I think it's fine to not pass the cancellation token here.